pkg/virtual/apiexport/controllers/apireconciler/apiexport_apireconciler_controller.go

```diff
@@ -149,7 +149,9 @@ func (c *APIReconciler) enqueueAPIResourceSchema(obj interface{}, logger logr.Lo
 		return
 	}
 
-	logger = logger.WithValues("cluster", clusterName, "name", name)
+	if logObj, ok := obj.(logging.Object); ok {
```

Comment:
We know it must be an APIResourceSchema, which is a logging object, but this works too :)